packages/@aws-cdk/aws-apigateway/lib/access-log.ts

```diff
@@ -0,0 +1,556 @@
+  }
+
+  /**
+   * The path to your resource.
+   * For example, for the non-proxy request URI of `https://{rest-api-id.execute-api.{region}.amazonaws.com/{stage}/root/child`,
+   * The $context.resourcePath value is `/root/child`.
+   * For more information: https://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-create-api-step-by-step.html
+   */
+  public static contextResourcePath() {
+    return '$context.resourcePath';
+  }
+
+  /**
+   * The deployment stage of the API request (for example, `Beta` or `Prod`).
+   */
+  public static contextStage() {
+    return '$context.stage';
+  }
+
+  /**
+   * The response received from AWS WAF: `WAF_ALLOW` or `WAF_BLOCK`. Will not be set if the stage is not associated with a web ACL.
+   * For more information: https://docs.aws.amazon.com/apigateway/latest/developerguide/apigateway-control-access-aws-waf.html
+   */
+  public static contextWafResponseCode() {
+    return '$context.wafResponseCode';
+  }
+
+  /**
+   * The complete ARN of the web ACL that is used to decide whether to allow or block the request.
+   * Will not be set if the stage is not associated with a web ACL.
+   * For more information: https://docs.aws.amazon.com/apigateway/latest/developerguide/apigateway-control-access-aws-waf.html
+   */
+  public static contextWebaclArn() {
+    return '$context.webaclArn';
+  }
+
+  /**
+   * The trace ID for the X-Ray trace.
+   * For more information: https://docs.aws.amazon.com/apigateway/latest/developerguide/apigateway-enabling-xray.html
+   */
+  public static contextXrayTraceId() {
+    return '$context.xrayTraceId';
+  }
+
+  /**
+   * The authorizer latency in ms.
+   */
+  public static contextAuthorizerIntegrationLatency() {
+    return '$context.authorizer.integrationLatency';
+  }
+
+  /**
+   * The integration latency in ms.
+   */
+  public static contextIntegrationLatency() {
+    return '$context.integrationLatency';
+  }
+
+  /**
+   * For Lambda proxy integration, this parameter represents the status code returned from AWS Lambda,
+   * not from the backend Lambda function.
+   */
+  public static contextIntegrationStatus() {
+    return '$context.integrationStatus.';
+  }
+
+  /**
+   * The response latency in ms.
+   */
+  public static contextResponseLatency() {
+    return '$context.responseLatency';
+  }
+
+  /**
+   * The response payload length.
+   */
+  public static contextResponseLength() {
+    return '$context.responseLength';
+  }
+
+  /**
+   * The method response status.
+   */
+  public static contextStatus() {
+    return '$context.status';
+  }
+}
+
+/**
+ * A API Gateway custom access log format.
+ */
+export interface IAccessLogFormat {
+  /**
+   * A API Gateway custom access log format
+   */
+  readonly format: string;
+}
+
+/**
+ * jsonWithStandardFields property
+ */
+export interface IJsonWithStandardFieldProps {
+  /**
+   * If this flag is enabled, $context.identity.sourceIp will be output to the log
```

Comment:
Prefer to use more natural language than $context.identity.sourceIp - "the source IP of the request" or something like that.

Reply:
Please address this comment.